apps/web/src/components/newsletters/NewsletterContent.tsx

```
@@ -0,0 +1,111 @@
+          </p>
+        );
+        currentParagraph = [];
+        paragraphKey++;
+      }
+    };
+
+    content.forEach((item, index) => {
+      switch (item.type) {
+        case "heading":
+          flushParagraph();
+          const HeadingTag = `h${item.level || 2}` as keyof JSX.IntrinsicElements;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)